src/main/resources/templates/fragments/userBar.html

```diff
@@ -9,15 +9,9 @@
   <nav class="content">
     <ul class="mobile-hidden" id="navigation">
       <li>
-        <ul class="mobile-hidden"
-            th:if="${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}">
+        <ul th:if="${T(uk.gov.ch.developer.docs.models.user.UserModel).isUserSignedIn(userModel)}"
+            th:remove="tag">
           <li class="user" id="signed-in-user" th:text="${userModel.getId() + ':'}"></li>
-          <li><a
-              class="piwik-event"
-              data-event-id="Your details - top menu" id="your-details"
-              th:href="@{{chsAccountUrl}/user/account(chsAccountUrl=${@environment.getProperty('account.url')})}">Your
-            details
-          </a></li>
           <li><a
```

Comment:
Please remove this link, when user is signed in the only information that is displayed is - User ID, Dev Forum, Sign out - in that order

Reply:
Okay sorry I didn't realise applications also had to go.